.github/workflows/autobuild.yaml

```diff
@@ -13,9 +13,11 @@ jobs:
     steps:
     - name: Find LLVM commit
       run: |
-        grep -m 1 -o "llvm-project@[[:xdigit:]]\{40,\}" << EOF | cut -f 2 -d@ > commit
-        ${{ github.event.release.body }}
+        perl -ne 'print "LLVM_REPO=$1\nLLVM_COMMIT=$2\n" if /(\w+)\/llvm-project@([[:xdigit:]]{4,})/' > commit.env <<EOF
```

Comment:
I think you want to capture all of `\w+/llvm-project`, not just the owner part (sorry!)